src/Render/D2L.Dev.Docs.Render.csproj

```diff
@@ -13,4 +13,10 @@
     <PackageReference Include="System.CommandLine.DragonFruit" Version="0.3.0-alpha.19577.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Templates\page.html">

```

Comment:
This could maybe become an embedded resource at some point

Reply:
Yeah, I didn't mean to include this in the PR, but I might as well update it

Reply:
Yep that's fine, either way.